service/scaladsl/kafka/server/src/test/scala/com/lightbend/lagom/scaladsl/kafka/broker/ScaladslKafkaApiSpec.scala

```diff
@@ -217,6 +215,25 @@ class ScaladslKafkaApiSpec extends WordSpecLike with Matchers with BeforeAndAfte
       for (i <- 1 to batchSize) queue.offer((i.toString, NoOffset))
       latch.await(10, TimeUnit.SECONDS) shouldBe true
```

Comment:
This test fails once every 20 runs on my local machine. It's hard to diagnose really but the failure is always that the `latch` was not opened. (see next comment)